File: Python/Interview041319.py
def alphabetical_order(arr):

    sorted_list = sorted(arr)
    for i in range(0, len(sorted_list)):
        print(sorted_list[i])


alphabetical_order(['Waltz', 'Tango', 'Viennese Waltz', 'Foxtrot',
                    'Cha Cha', 'Samba', 'Rumba', 'Paso Doble', 'Jive'])

File: JS/Interview041319.js
const alphabetic_order = (arr) => {
    sorted = arr.sort()
    for (i = 0; i < sorted.length; i++) {
        console.log(sorted[i])
    }
}

alphabetic_order(['Waltz', 'Tango', 'Viennese Waltz', 'Foxtrot',
    'Cha Cha', 'Samba', 'Rumba', 'Paso Doble', 'Jive'])